lerna-package-lock.json

```diff
@@ -42258,6 +42255,3692 @@
 			"resolved": "https://registry.npmjs.org/notepack.io/-/notepack.io-2.3.0.tgz",
 			"integrity": "sha512-9RiFDxeydHsWOqdthRUck2Kd4UW2NzVd2xxOulZiQ9mvge6ElsHXLpwD3HEJyql6sFEnEn/eMO7HSdS0M5mWkA=="
 		},
+		"npm": {
```

Comment:
I don't think we want to list npm as a dev dependency, we simply want to do a global install of it on the system.